app/services/stock_info_service.py

```
@@ -589,4 +624,40 @@
+        print(f"    - 주문 수량: {quantity:.8f}")
+        print(f"    - 예상 금액: {format_decimal(estimated_amount, '₩')}원")
+
+        # 지정가 매수 주문
+        order_result = await upbit.place_buy_order(
+            market=market,
+            price=str(adjusted_price),
+            volume=str(quantity),
+            ord_type="limit"
+        )
+
+        print(f"    ✅ 주문 성공:")
```

Ruff (0.14.6)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)